src/coreclr/dlls/mscoree/exports.cpp

```diff
@@ -130,6 +147,31 @@ static void InitializeStartupFlags(STARTUP_FLAGS* startupFlagsRef)
     *startupFlagsRef = startupFlags;
 }
 
+#if defined(TARGET_ANDROID)
+static void ConvertConfigPropertiesToUnicode(
```

Comment:
Why is this version needed?

Reply:
The Android version is different enough that I thought it would make it clearer to put it in a separate version of the function, and keep the name for familiarity.